api/src/test/java/org/openmrs/util/databasechange/Database1_9_7UpgradeIT.java

```diff
@@ -503,9 +511,63 @@ private boolean roleHasPrivilege(String role, String privilege) {
 
 	@Test
 	public void shouldUpgradeLiquibase() throws IOException, SQLException {
-		upgradeTestUtil.upgrade("liquibase-update-to-latest.xml");
+		upgradeTestUtil.upgrade( LIQUIBASE_UPDATE_TO_LATEST_XML );
 
 		// no explicit assertions here, this test serves to see whether the master is executed without raising an exception.
 
 	}
+	
+	@Test
+	public void shouldFindUnrunDatabaseChanges() throws IOException, SQLException {
+		// given
+		//
```

Comment:
I find the above comment redundant?

Reply:
The comment is supposed to clearly differentiate between preparing the test case and actually executing it. Admittedly "// when" and "// then" are very short so that is not needed here.